codecov.yml

```diff
@@ -4,6 +4,3 @@ coverage:
       default:
         target: auto     # auto compares coverage to the previous base commit
         threshold: 0.5%  # coverage can drop by up to 0.5% while still posting success
-    patch:
-      default:
-        informational: true
```

Comment:
Yeah, kinda glad this is now gone... never quite realised that it snuck in

Just a wee bit noisy for my personal liking :wink: